package.json

```diff
@@ -3,6 +3,13 @@
   "version": "1.0.0-a.14",
   "description": "High-performance spatial analysis in a web browser, Node.js, and reproducible execution across programming languages and hardware architectures.",
   "main": "./dist/index.js",
+  "exports": {
+    ".": {
```

Comment:
This will prevent to import other modules individually. If I'm not wrong they should all be listed here

Reply:
Yes, my understanding also is that this will prevent the import of other modules individually. However, all exposed modules should be available through the index. Adding all modules would add a maintainability headache. From the other examples I looked at:

- https://unpkg.com/browse/preact@10.6.6/package.json
- https://unpkg.com/browse/nanoid@3.3.1/package.json

only the package index's are exposed. Is there really a need to list all the modules?

Reply:
I'm from mobile so I can't look for the exact name, but in our case we need the worker pool utility that AFAIK is not exposed from the index module

Reply:
Ah, ok. The worker pool should be exposed as

```
import { WorkerPool } from 'itk-wasm'
```

per:

https://github.com/InsightSoftwareConsortium/itk-wasm/blob/6539760e0d0a70eba265c6dd2b661331cf15a4ef/src/core/index.ts#L35-L38

https://github.com/InsightSoftwareConsortium/itk-wasm/blob/6539760e0d0a70eba265c6dd2b661331cf15a4ef/src/index.ts#L3

If not, let's fix it and make sure it meets your needs.

Reply:
I'll double check tomorrow, thanks!

Reply:
It works, thanks 🙇